heroku.yml

```
@@ -0,0 +1,3 @@
+build:
```

```suggestion
setup:
  config:
    BPFS_STORAGE: database
build:
```
I tried hiding the BPFS_STORAGE textbox in Heroku one click menu by putting it here instead of app.json. It doesn't work. The documentation says that the setup section is in beta, so maybe it's not supported.